### PR TITLE
main/ldb: upgrade to 1.4.3

### PR DESCRIPTION
**Pull request by J0WI**
Required for samba 4.9

### CODE REVIEW

File: main/ldb/APKBUILD [hunk 1]
```diff
@@ -1,16 +1,15 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=ldb
-pkgver=1.3.6
+pkgver=1.4.3
```
[the remark this answers is not in the document]
Comment (J0WI) [reply main/ldb/APKBUILD]: 1.4.3 is required by samba, 1.5 is not yet supported.

**Comment by J0WI** [comment]
Remark: samba 4.8 requires ldb 1.3.6

**Comment by tcely** [comment]
#5932 has these changes already, correct?

**Comment by J0WI** [comment]
#5932 updates to the same version, but doesn't cover all changes. I had no issues with the checks here.

**Comment by tcely** [comment]
@J0WI I think #5932 now covers these changes. Do you agree?

**Comment by J0WI** [comment]
There is still an open point in https://github.com/alpinelinux/aports/pull/5932#issuecomment-454098390, but I think we can now close this in favor of #5932 .

**Pull request closed without merging**